src/Stratis.Bitcoin/Broadcasting/BroadcasterManagerBase.cs

```diff
@@ -58,5 +60,25 @@ public void AddOrUpdate(Transaction transaction, State state)
         }
 
         public abstract Task<Success> TryBroadcastAsync(Transaction transaction);
+
+        protected void PropagateTransactionToPeers(Transaction transaction, bool skipHalfOfThePeers = false)
```

Comment:
Can you please add some documentation on these methods?

Reply:
The FullNodeBroadcasterManager is also broadcasting to half the peers, so calling this method with `skipHalfOfThePeers = false` is a change of functionality.
Both should call only broadcast to half the peers, for now, so we don't need the parameter.

Reply:
>Both should call only broadcast to half the peers, for now, so we don't need the parameter.

In current implementation of broadcasting algorithm for light node it is used (on light node we send tx to half of the peers and expect other half to propagate it back to us. that's how we know that our tx is valid and was propagated).

Reply:
I can see that in the current implementation, the FullNodeBroadcasterManager is also broadcasting to half the peers. Are yo unot seeing the same thing?

Reply:
Ah, yes. 
I thought that it was a logical error and writer did not mean to do that. 
I'm removing my change because it's refactoring. Maybe I should introduce it in later PRs